README.md

```diff
@@ -2015,6 +2015,7 @@ I've documented some recommendations for writing a new plugin [here](https://git
 ### ZSH Tools
 
 * [ShellSpec](https://github.com/shellspec/shellspec) - A full-featured BDD unit testing framework for dash, bash, ksh, ZSH and all POSIX shells.
+* [zsh-bench](https://github.com/romkatv/zsh-bench) - A benchmark for interactive ZSH
```

Comment:
Please add a `.` on the end of the description, otherwise looks good

Reply:
Done.

Two out of tree existing elements in this list don't have trailing periods, so I went with the majority. Now we have a nice 50/50 split \o/

Reply:
I'll fix the others, thanks for the heads up